client/src/components/Card/Card.css

```diff
@@ -0,0 +1,34 @@
+.card {
+	height: 169px;
+	width: 234px;
```

Comment:
We may not want and absolute height and width for this - the sizing may need to be determined by the contents or the context